# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/systemd/hotfix-lib64-warn.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- systemd-258.3/src/shared/base-filesystem.c.vanilla	2026-02-27 13:27:47.328679290 +0100
+++ systemd-258.3/src/shared/base-filesystem.c	2026-02-27 13:31:12.391300837 +0100
@@ -134,12 +134,9 @@
         /* s390-linux-gnu */
 #elif defined(__sparc__)
 #endif
-        /* gcc doesn't allow pragma to be used within constructs, hence log about this separately below */
+#warning "Please add an entry above specifying whether your architecture uses /lib64/, /lib32/, or no such links."
 };
 
-#ifndef KNOW_LIB64_DIRS
-#  pragma message "Please add an entry above specifying whether your architecture uses /lib64/, /lib32/, or no such links."
-#endif
 
 int base_filesystem_create_fd(int fd, const char *root, uid_t uid, gid_t gid) {
         bool empty_fs = false;
